python/private/pypi/versions.bzl

```diff
@@ -0,0 +1,14 @@
+"""Version and integrity information for downloaded artifacts"""
+
+# From: https://github.com/astral-sh/uv/releases
+UV_TOOL_VERSIONS = {
```

Comment:
I think that this could be the default value in the `uv` dependency extension and we can override the value by supplying these arguments to it from a outside rules_python, like:
```starlark
uv = use_extension("@rules_python//python/extensions:uv.bzl")

uv.download(
    version = "0.2.13",
    files = {
        "aarch64-apple-darwin": "deadbeef",
        ...
    },
    base_url = "if I really want to do it like this it would be possible, but experimental downloader config is nicer?",
)
```

Reply:
Yes, I don't think that I would keep this growing for eternity. Honestly, probably just retain a single version at a time in the rules. I think it might just be useful to have that extra version discriminator, even if just for brining along a version number into the toolchain for tracking purposes.